test/CSModule.t.sol

```diff
@@ -1650,282 +1650,6 @@ contract CSMObtainDepositData is CSMCommon {
     }
 }
 
-contract CSMClaimRewards is CSMCommon {
```

Comment:
Do we need to extend tests for claim methods in the accounting tests? Like assert that depositable had changed.

Reply:
yup I can check that there was a call to the CSM mock with no side effects